internal/daysteps/daysteps.go

```diff
@@ -10,6 +16,24 @@ var (
 
 func parsePackage(data string) (int, time.Duration, error) {
 	// ваш код ниже
+	parsedString := strings.Split(data, ",")
+	if len(parsedString) != 2 {
+		return 0, 0, errors.New("incorrect string format")
+	}
+	steps, err := strconv.Atoi(parsedString[0])
+	if err != nil {
+		return 0, 0, fmt.Errorf("convertation error: %v", err)
```

Comment:
При оборачивании нужно использовать `%w` (wrap), а просто при выводе в консоль `%v`
```suggestion
		return 0, 0, fmt.Errorf("convertation error: %w", err)
```

Reply:
Нужно везде поправить

Reply:
везде поправил